server/api/routes/ssotRoutes.test.js

```diff
@@ -553,3 +553,61 @@ describe('ssot/delete/:robotId', () => {
     expect(foundJobs.length).toBe(0);
   });
 });
+
+describe('/deleteParameters', () => {
+  it('deletes removed activity related parameter', async () => {
+    await dbLoader.loadSsotInDb();
+    await dbLoader.loadParametersInDb();
+
+    let deletedActivityList = [
+      testSsot.elements[2].id,
+      testSsot.elements[3].id,
+    ];
+    deletedActivityList = JSON.stringify(deletedActivityList);
+
+    const request = httpMocks.createRequest({
+      method: 'DELETE',
+      query: {
+        activityIdList: deletedActivityList,
+        robotId: testRobotId,
+      },
+    });
+    const response = httpMocks.createResponse();
+
+    await ssotVariableController.deleteMany(request, response);
+
+    const foundParameters = await mongoose.model('parameter').find().exec();
+    expect(foundParameters.length).toBe(1);
```

Comment:
A nice alternative solution for less magic numbers could have been 
```javascript
expect(foundParameters.length).toBe(testSsot.elements.length - deletedActivityList.length); 
```
but I do not see that as that improtant